pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -1309,3 +1332,90 @@ func initializeAnnotations(cd *hivev1.ClusterDeployment) {
 		cd.Annotations = map[string]string{}
 	}
 }
+
+// managePullSecrets() takes pull secretes in to account and returns the merged pull request
```

Comment:
Don't append the `()` after the function name in the comment block.

Reply:
I am just updating the TOC which was broken before. I can move it to a different commit. But it is very small change and only doc changes. Hence thought of keeping it in this PR